srv_fai_config/files/etc/default/grub.d/20_efi_ipv6.cfg/SEAPATH_COMMON

```diff
@@ -1 +1 @@
-GRUB_CMDLINE_LINUX_DEFAULT=" efi=runtime ipv6.disable=1"
+GRUB_CMDLINE_LINUX_DEFAULT=" efi=runtime ipv6.disable=1 console=ttyS0,115200"
```

Comment:
I am not sure this is the best place.
I preferred using a dedicated file like: srv_fai_config/files/etc/default/grub.d/30_console.cfg/SEAPATH_COMMON
It is possible to reuse the GRUB_CMDLINE_LINUX_DEFAULT variable. Also if you plan to use this modification for other image add console=tty0 for keeping the console on display.
GRUB_CMDLINE_LINUX_DEFAULT="$GRUB_CMDLINE_LINUX_DEFAULT console=ttyS0,115200 console=tty0"